test/ProjectTemplates/Microsoft.Extensions.AI.Templates.IntegrationTests/TemplateSandbox/nuget.template_test.config

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <!--
+    This file will get copied into each test project directory to control which NuGet sources
+    get used during project restore.
+  -->
+
+  <fallbackPackageFolders>
+    <!-- Clear to avoid inheriting from the repo root NuGet.config -->
+    <clear />
+  </fallbackPackageFolders>
+  <packageSources>
+    <clear />
+    <!-- 'LOCAL_SHIPPING_PATH' is an environment variable that gets set by the template tests. -->
+    <add key="local-shipping" value="%LOCAL_SHIPPING_PATH%" />
+    <add key="nuget" value="https://api.nuget.org/v3/index.json" />
```

Comment:
For scenarios where AI packages and/or non-AI packages from this repo are set to use pinned version from nuget.org, will this setup ensure the pinned versions are resolved instead of the local-shipping versions?

For my own local testing, I've been deleting the locally built packages from the folder for anything that I want to ensure gets resolved from a pinned version to be certain the pinned version is resolved and used.

Reply:
From my testing, if I switch to using the pinned versions of the template, then the pinned versions get resolved over the local ones (even if the local version is "higher" than the pinned one).

However, I just tried testing what would happen if the specified pinned version could not be found (causing a restore warning), and while the warning gets logged to the output, it doesn't get logged to standard error and the test still passes. Going to push a fix for that since restore warnings should be considered failures.

Reply:
Fixed now!